frontend/src/components/OdhAppCard.tsx

```diff
@@ -100,14 +100,16 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
         />
       </CardHeader>
       <CardTitle>
-        {odhApp.spec.displayName}
-        <Tooltip content="Red Hat Certified and Supported">
-          <img
-            className={supportedImageClasses}
-            src="../images/CheckStar.svg"
-            alt="Red Hat Certified and Supported"
-          />
-        </Tooltip>
+        <span className="odh-card__title-supported">
+          {odhApp.spec.displayName}
+          <Tooltip content="Red Hat Certified and Supported">
```

Comment:
should this be sentence case?

Reply:
Good question! I'm checking with those that know better than me, and will reply with the answer.

Reply:
Pretty sure we should do sentence case, but I'm waiting to hear back from OCM to see what they do.

Reply:
Sentence case it is, thanks for spotting that @jschuler!